lib/ansible/module_utils/vmware.py

```diff
@@ -1382,3 +1382,12 @@ def to_json(self, obj, properties=None):
         else:
             result = self._jsonify(obj)
         return result
+
+    def get_folder_path(self, cur):
```

Comment:
Can we use [`get_vm_path`](https://github.com/ansible/ansible/blob/7191d4588563107a19def557dc4b439e01c80fda/lib/ansible/module_utils/vmware.py#L973) ?

Reply:
Yes probably indeed.

Reply:
Well, no. It's two different things. However, it would make sense to move get_folder_path in `module_utils`.